staging/src/k8s.io/cli-runtime/pkg/resource/builder.go

```diff
@@ -74,6 +74,7 @@ type Builder struct {
 
 	paths  []Visitor
 	stream bool
+	stdin  bool
```

Comment:
Unclear what this means when looking at it out of context.

```suggestion
	stdinInUse  bool
```